src/pages/FrenchFamilyBenefitsExample.res

```diff
@@ -275,15 +309,36 @@ module Simulator = {
             <div className=%twc("flex flex-col mx-4")>
               <Input.CheckBox
                 label={<Lang.String
-                  english="Rights open before 2021" french=`Droits ouverts avant 2012`
+                  english="Rights open before 2012" french=`Droits ouverts avant 2012`
                 />}
                 onChange={_ => {
                   updateCurrentState({
                     ...allocFamInput,
-                    avaitEnfantAChargeAvant1erJanvier2012: switch allocFamInput.avaitEnfantAChargeAvant1erJanvier2012 {
-                    | None | Some(false) => Some(true)
-                    | Some(true) => Some(false)
-                    },
+                    avaitEnfantAChargeAvant1erJanvier2012: allocFamInput.avaitEnfantAChargeAvant1erJanvier2012->updateOptionalBool,
+                  })
+                }}
+              />
+              <Input.CheckBox
+                label={<Lang.String
+                  english="The person who has the actual and permanent charge is a parent"
+                  french=`La personne a la charge effective et permanente est parent`
+                />}
+                onChange={_ => {
+                  updateCurrentState({
+                    ...allocFamInput,
+                    personneChargeEffectivePermanenteEstParent: allocFamInput.personneChargeEffectivePermanenteEstParent->updateOptionalBool,
+                  })
+                }}
+              />
+              <Input.CheckBox
+                label={<Lang.String
+                  english="The person who has the actual and permanent charge fulfills the title II"
+                  french=`La personne a la charge effective et permanente remplit le Titre II`
```

Comment:
It's the title I and not the title II. You should put a link to https://www.legifrance.gouv.fr/codes/section_lc/LEGITEXT000006073189/LEGISCTA000006141653/#LEGISCTA000006141653 in this input label.

Reply:
Done!